api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/ARVsDataConverter.java

```diff
@@ -38,10 +38,10 @@ else  if(arvResultsConcept != null && arvResultsConcept.equals(Dictionary.getCon
         }
 
         else  if(arvResultsConcept != null && arvResultsConcept.equals(Dictionary.getConcept("2c000b41-f9d7-40c1-8de0-bce91dbae932"))){
```

Comment:
@solemabrothers Why not add a check for arvResultsConcept being null, so that you simplify the checks, see NationalityPersonAttribute example